### DISCUSSION TITLE
relational collection filter in other collection

### DISCUSSION THREAD

**Discussion by ckoriya** [directus/directus#8262] [Archive]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues) that already describes my problem.

### Describe the Bug

I have three collections, Users,States,District.

in District there is data with State wise means In District table there is stateid,districtid and district name.

In Users there is stateid and districtid when i select stateid i wants to filter district as per stateid currently its showing me all state wise all district ,

so please help me how can i filter state wise district in directus users collection.
 

### To Reproduce

I have three collections, Users,States,District.

in District there is data with State wise means In District table there is stateid,districtid and district name.

In Users there is stateid and districtid when i select stateid i wants to filter district as per stateid currently its showing me all state wise all district ,

so please help me how can i filter state wise district in directus users collection.
 

### What version of Directus are you using?

9.0.0

### What version of Node.js are you using?

latest

### What database are you using?

postgresql

### What browser are you using?

Chrome

### What operating system are you using?

Windows

### How are you deploying Directus?

locally

**Comment by azrikahar** [comment]
Seems like #7080 fits the use case here
